src/server.h

```diff
@@ -3503,10 +3501,9 @@ sds hashTypeCurrentObjectNewSds(hashTypeIterator *hi, int what);
 hfield hashTypeCurrentObjectNewHfield(hashTypeIterator *hi);
 int hashTypeGetValueObject(redisDb *db, kvobj *kv, sds field, int hfeFlags,
                            robj **val, uint64_t *expireTime, int *isHashDeleted);
-int hashTypeSet(redisDb *db, robj *o, sds field, sds value, int flags);
+int hashTypeSet(redisDb *db, kvobj *kv, sds field, sds value, int flags);
 robj *hashTypeDup(kvobj *kv, uint64_t *minHashExpire);
-uint64_t hashTypeRemoveFromExpires(ebuckets *hexpires, robj *o);
-void hashTypeAddToExpires(redisDb *db, kvobj *hashObj, uint64_t expireTime);
+uint64_t hashTypeActiveExpire(kvobj *o, redisDb *db, uint32_t *quota, int updateSubexpires);
```

Comment:
maybe `db` can be the first parameter, like before
```suggestion
uint64_t hashTypeActiveExpire(redisDb *db, kvobj *kv, uint32_t *quota, int updateSubexpires);
```